notebooks/official/tabnet/sdk2_remote_tabnet_training.ipynb

```diff
@@ -0,0 +1,693 @@
+{
```

Comment:
<p>"Train a the model remotely" &gt; "Train a model remotely"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
done